server/dev/server.go

```diff
@@ -69,10 +70,13 @@ func main() {
 	if err != nil {
 		server.BootLogFatalf("reading config: %v", err)
 	}
-	logger := logger.NewSimpleLogger(os.Stderr, "debug")
+	logger := logger.NewSimpleLogger(os.Stderr, "info")
```

Comment:
Why this change?

Reply:
Push server logs otherwise IP addresses and tokens and the board decided that we do not want to have this information in the logs for privacy reason. For debugging purposes during the first weeks it might be necessary to turn it on again. Can we switch this dynamically during runtime with Go?

Reply:
I'm a bit ignorant on Go, I believe that there should be some way to control this via environment variables, but I don't know.
I think the reason you mentioned is very valid, so feel free to keep your change the way it is.